pylhc_submitter/submitter/htc_utils.py

```diff
@@ -15,35 +15,35 @@
 import logging
 import subprocess
 from pathlib import Path
-from typing import Union
+from typing import Any, Dict, List, Union
 
 from pandas import DataFrame
 
-from pylhc_submitter.utils.environment_tools import on_windows
+from pylhc_submitter.constants.job_submitter import (COLUMN_DEST_DIRECTORY, COLUMN_JOB_DIRECTORY,
+                                                     COLUMN_JOB_FILE, COLUMN_SHELL_SCRIPT,
+                                                     EXECUTEABLEPATH, NON_PARAMETER_COLUMNS)
+from pylhc_submitter.submitter.iotools import is_eos_uri
+from pylhc_submitter.submitter.mask import is_mask_file
+from pylhc_submitter.utils.environment import on_windows
 
 try:
     import htcondor
 except ImportError:  # will be handled by job_submitter
-    pass
+    class htcondor:
+        """Dummy HTCondor module. To satisfy the typing. """
+        Submit: Any = None
 
-from pylhc_submitter.constants.external_paths import MADX_BIN, PYTHON2_BIN, PYTHON3_BIN
 
 LOG = logging.getLogger(__name__)
 
+# HTC Constants ################################################################
 
 SHEBANG = "#!/bin/bash"
 SUBFILE = "queuehtc.sub"
 BASH_FILENAME = "Job"
 
 HTCONDOR_JOBLIMIT = 100000
 
-EXECUTEABLEPATH = {
-    "madx": MADX_BIN,
-    "python3": PYTHON3_BIN,
-    "python2": PYTHON2_BIN,
-}
-
-
 CMD_SUBMIT = "condor_submit"
 JOBFLAVOURS = (

```

Comment:
Should we take the opportunity to extract these constants to something like `pylhc_submitter.constants.htcondor`?